### DISCUSSION TITLE
Nanopore Iso-Seq without Illumina

### DISCUSSION THREAD

**Discussion by francicco** [ablab/spades#1148] [Q&A]
### Description of bug

Hi,

I have ~50M reads of Iso-Seq Nanopore data, but I don't have illumina RNAseq. I'd like to assemble the reads and map the transcripts to the genome to perform the genome annotation.

My understanding is that rnaSpades only perform a hybrid approach. 
Is that correct or there's anything I'm missing here?

Thanks a lot
F

### spades.log

Don't have it

### params.txt

Don't know

### SPAdes version

3.15

### Operating System

CentOS

### Python Version

Phython 3

### Method of SPAdes installation

binaries

### No errors reported in spades.log

- [X] Yes

**Comment by andrewprzh** [comment]
Dear @francicco 

You are right, rnaSPAdes only supports hybrid approach.
Since you have a genome, I would recommend you to use a reference-based tool, for example [IsoQuant](https://github.com/ablab/IsoQuant). It will generate a GTF annotation right from your Nanopore data.

Best
Andrey

**Comment by francicco** [comment]
Hi @andrewprzh,

That is a great tip! Thanks a lot.
Best
F
